treeherder/etl/perf_data_adapters.py

```diff
@@ -90,16 +90,21 @@ def _round(num):
         return round(num, 2)
 
     @staticmethod
-    def _calculate_summary_data(job_id, result_set_id, push_timestamp, results):
+    def _extract_summary_data(suite_data, summary):
+        suite_data["geomean"] = summary["suite"]
```

Comment:
We should file a followup bug to change the terminology here, since the suite summary value is not always a geometric mean. Maybe we should just say "value".